src/pmdas/llc/GNUmakefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2023 Red Hat.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+
+TOPDIR = ../../..
+include $(TOPDIR)/src/include/builddefs
+
+IAM	= llc
+DOMAIN	= LLC
+CMDTARGET = pmda$(IAM)$(EXECSUFFIX)
+LIBTARGET = pmda_$(IAM).$(DSOSUFFIX)
+PMDAINIT = $(IAM)_init
+
+PMDAADMDIR = $(PCP_PMDASADM_DIR)/$(IAM)
+PMDATMPDIR = $(PCP_PMDAS_DIR)/$(IAM)
+PMDACONFIG = $(PCP_SYSCONF_DIR)/$(IAM)
+
+LLDLIBS = $(PCP_PMDALIB)
+
+CFILES	= llc.c
+DFILES	= help
+
+LDIRT	= domain.h *.o $(IAM).log $(CMDTARGET) $(LIBTARGET)
+
+MAN_SECTION = 1
+MAN_PAGES = pmda$(IAM).$(MAN_SECTION)
+MAN_DEST = $(PCP_MAN_DIR)/man$(MAN_SECTION)
+
+LCFLAGS = -I.
+CFLAGS_OPT = -g
+
+default_pcp default:	build-me
+
+include $(BUILDRULES)
+
+build-me:	$(CMDTARGET) $(LIBTARGET)
+
+install_pcp install:	default 
+	$(INSTALL) -m 755 -d $(PMDAADMDIR)
+	$(INSTALL) -m 755 -d $(PMDATMPDIR)
+	$(INSTALL) -m 755 -t $(PMDATMPDIR) Install Remove $(CMDTARGET) $(LIBTARGET) $(PMDAADMDIR)
+	$(INSTALL) -m 644 -t $(PMDATMPDIR) root pmns domain.h $(DFILES) $(PMDAADMDIR)
+	@$(INSTALL_MAN)
```

Comment:
Can we install a systemd mount file here too to make persistence easier for users?

https://www.freedesktop.org/software/systemd/man/systemd.mount.html
https://www.redhat.com/en/blog/performance-tuning-at-the-edge
https://github.com/intel/intel-cmt-cat/wiki/resctrl

... the different options might be something we can handle via ConditionCPUFeature= ... ?
https://www.freedesktop.org/software/systemd/man/systemd.unit.html